Documentation/ceph-object-store-user-crd.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Ceph Object Store User
+weight: 34
```

Comment:
The `weight` determines where in the table of contents the topic will show up. I'd recommend changing this to `35` so it will be after the object store CRD. You'll need to change the weight in ceph-filesystem-crd.md to `36`.